ember-cli-build.js

```diff
@@ -2,48 +2,11 @@
 
 const EmberApp = require('ember-cli/lib/broccoli/ember-app');
 
-const environment = process.env.EMBER_ENV;
-const IS_PROD = environment === 'production';
-const IS_TEST = environment === 'test';
-
 module.exports = function (defaults) {
   let app = new EmberApp(defaults, {
-    autoImport: {
-      exclude: ['flandersMunicipalities.ts'],
```

Comment:
This doesn't seem to be imported anywhere? Outdated config I guess?

Reply:
I guess it was use before using municipality from the server